mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml

```diff
@@ -284,15 +284,17 @@
 
   <t>
     d-CW and and S-Labels are used as defined in <xref
-    target="I-D.ietf-detnet-mpls"/> and are not modified by this document.
+    target="I-D.ietf-detnet-mpls"/> and are not modified by this document. 

```

Comment:
Change figure Add F-labels below S-label and text should say "zero or more F-label(s)